h/emails/util.py

```diff
@@ -8,3 +8,7 @@ def get_user_url(user: User, request: Request) -> str | None:
         return request.route_url("stream.user_query", user=user.username)
 
     return None
+
+
+def email_subject(document: str) -> str:
+    return f"New activity on {document}"
```

Comment:
The slack thead uses `in` instead of `on`. Not sure which one is correct.

Reply:
I'm trusting someone who actually speaks English here 😅 

https://github.com/hypothesis/h/pull/9432#discussion_r2018303113

Reply:
ah! I guess it's different for assignments? :shrug: